packages/lib-grommet-theme/src/index.js

```diff
@@ -132,7 +135,14 @@ const theme = deepFreeze({
       small: `20px`,
       medium: `30px`,
       large: `50px`,
-      xlarge: `90px`
+      xlarge: `90px`,
+
+      'xxsmall-neg': '-5px',
+      'xsmall-neg': `-10px`,
+      'small-neg': `-20px`,
+      'medium-neg': `-30px`,
+      'large-neg': `-50px`,
+      'xlarge-neg': `-90px`
```

Comment:
I've published the grommet theme recently because it is being used with the text editor app Will is working on. This PR doesn't need to publish this update, but can you open an issue so we know it needs to be done.

Reply:
https://github.com/zooniverse/front-end-monorepo/issues/1111